src/deploy/functions/release/sourceTokenScraper.ts

```diff
@@ -1,5 +1,10 @@
 import { logger } from "../../../logger";
-import { Timer } from "./timer";
+
+enum TokenState {
+  "NONE",
+  "VALID",
+  "INVALID",
+}
```

Comment:
tangent: enums are controversial in TS for some reason.

https://blog.logrocket.com/why-typescript-enums-suck/

Internally, I think we prefer string literals:

```
type TokenState = "none" | "valid" | "invalid"
```